components/QuestionForm.js

```diff
@@ -95,7 +98,17 @@ class QuestionForm extends Component {
 		let email = this.refs.mail_address.value.trim();
 		let text = this.state.message.trim();
 
-		if(!email || !text){
+		if( !text || !email || !email.match(/^[\w\-\+_]+(?:\.[\w\-\+_]+)*\@[\w\-\+_]+\.[\w\-\+_]+(?:\.[\w‌​\-\+_]+)*\s*$/) ){
```

Comment:
mmmmmm not thrilled about having the same check twice, can't we check the email first (have the return inside), then check the text (have the return inside too there)